rasa/nlu/selectors/response_selector.py

```diff
@@ -262,9 +267,9 @@ def _create_retrieval_intent_mapping(
 
         retrieval_intent_mapping = {}
         for example in training_data.intent_examples:
```

Comment:
should this rather loop over `training_data.response_examples`? otherwise there is always going to be 
`None: None` entry in this dictionary